README.md

```diff
@@ -37,25 +40,38 @@ CMakeToolchain
 cmake_layout
 
 ```
-**NOTE:** If using conan version 1.59 Ensure that the conan profile is configured to use ABI 11 (libstdc++11: New ABI.) standards according to https://docs.conan.io/en/1.60/howtos/manage_gcc_abi.html
 
-### Building locally 
-```
-$ cd up-cpp-client-zenoh
-$ mkdir build
-$ cd build
-$ conan install ../ 
-$ cmake ../ -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release
-$ cmake --build . --config Release -- -j
-```
+**NOTE:** If using conan version 1.59 Ensure that the conan profile is
+configured to use ABI 11 (libstdc++11: New ABI) standards according to
+https://docs.conan.io/en/1.60/howtos/manage_gcc_abi.html
 
-#### Creating conan package locally 
+## Building locally
+
+The following steps are only required for developers to locally build and test
+up-transport-zenoh-cpp, If you are making a project that uses
```

Comment:
Are we suggesting the project get renamed from up-client to up-transport? Might be a good time to do so actually.

Reply:
I have been _very subtly_ adding that change in all the documentation. It seems like the right direction since these libraries _should_ only be implementing the L1 transport.

Reply:
I'm tracking it on the 1.5.8 board as https://github.com/eclipse-uprotocol/up-client-zenoh-cpp/issues/52 and https://github.com/eclipse-uprotocol/up-client-vsomeip-cpp/issues/6